nifi-nar-bundles/nifi-standard-services/nifi-oauth2-provider-api/src/main/java/org/apache/nifi/oauth2/AccessDetails.java

```diff
@@ -17,53 +17,64 @@
 
 package org.apache.nifi.oauth2;
 
-public class AccessToken {
+public class AccessDetails {
     private String accessToken;
     private String refreshToken;
     private String tokenType;
-    private Integer expires;
-    private String scope;
+    private Integer expiresIn;
```

Comment:
This should be changed to an `int` to avoid unexpected NullPointerExceptions. Changing this to `java.time.Duration` would be a helpful improvement to avoid potential ambiguity.